_global/css/adversary.css

```diff
@@ -59,6 +59,21 @@ adversary-title{
   width:140px;
 }
 
+adversary-base-dif{
+  display:block;
+  position:absolute;
+  right:207px;
+  top:45px;
+  height:44px;
+  width:600px;
+  
+  text-align:right;
+  line-height: 36px;
+  font-size:16px;
+  font-family: 'DK Snemand';
+  text-transform: uppercase;
+}
+
```

Comment:
This positions the Base Difficulty text. I didn't really put much thought into the height/width so if folks see any concerns there, please edit.